README.md

```diff
@@ -39,10 +45,12 @@ mkdir -p /tmp/cannon
 minigeth/go-ethereum 13284469
 
 # write out the golden MIPS minigeth start state
-mipsevm/mipsevm
+yarn
+(cd mipsevm && ./evm.sh)
+./mipsevm/mipsevm
 
 # generate MIPS checkpoints for 13284469 -> 13284470
-mipsevm/mipsevm 13284469
+./mipsevm/mipsevm 13284469
```

Comment:
The dot in front when there is a dir is not strictly necessary. Let's drop it for consistency?